components/ContractViewer/ContractTreeView.tsx

```diff
@@ -0,0 +1,49 @@
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { TreeView } from '@mui/x-tree-view/TreeView'
+
+import { ContractInfo } from './ContractState'
+import ContractTreeNode from './ContractTreeNode'
+
+type ContractTreeViewProps = {
+  forest: any[]
```

Comment:
`any` disables type checks completely, which is bad practice. Maybe you can update this to have proper type check in place.